src/sqlparser/src/parser.rs

```diff
@@ -2129,11 +2129,27 @@ impl Parser<'_> {
             }
         }
 
+        let barrier_interval_ms = if self.parse_word("BARRIER_INTERVAL_MS") {
```

Comment:
Does it mean users should write the options in the strict order as [OWNER|RESOURCE_GROUP] -> BARRIER_INTERVAL_MS -> CHECKPOINT_FREQUENCY? 

Doesn't look like a good design. Can we support specifying the config in arbitary order?